modules/osd-create-cluster-red-hat-account.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * osd_quickstart/osd-quickstart.adoc
+// * osd_cluster_create/creating-an-aws-cluster.adoc
+// * osd_cluster_create/creating-a-gcp-cluster.adoc
+
+ifeval::["{context}" == "osd-getting-started"]
+:osd-quickstart:
+endif::[]
+ifeval::["{context}" == "osd-creating-a-cluster-on-aws"]
+:osd-on-aws:
+endif::[]
+ifeval::["{context}" == "osd-creating-a-cluster-on-gcp"]
+:osd-on-gcp:
+endif::[]
+
+:_content-type: PROCEDURE
+ifdef::osd-quickstart[]
+[id="create-cluster-red-hat-account_{context}"]
+= Creating a cluster with a Red Hat cloud account
+endif::osd-quickstart[]
+ifdef::osd-on-aws[]
+[id="osd-create-aws-cluster-red-hat-account_{context}"]
+= Creating a cluster on AWS with a Red Hat cloud account
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+[id="osd-create-aws-cluster-ccs_{context}"]
+= Creating a cluster on GCP with a Red Hat cloud account
+endif::osd-on-gcp[]
+
+Through {OCM}, you can create an {product-title} cluster 
+ifdef::osd-on-aws[]
+on {AWS} 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+on {GCP} 
+endif::osd-on-gcp[]
+using a standard cloud provider account owned by Red Hat.
+
+.Procedure
+
+. Log in to {console-redhat-com} and click *Create Cluster*.
+
+. In the *Cloud* tab, click *Create Cluster* in the *Red Hat OpenShift Dedicated* row.
+
+. In the *Billing model* dialog, configure the subscription type and infrastructure type:
+.. Select a subscription type.
```

Comment:
`Red Hat cloud account` only support subscription type `Annual` so it's better to clarify in the `Select a subscription type` step

Reply:
In the stage environment, I see two available subscription options for the Red Hat cloud account infrastructure type:

- Free trial (upgradeable)
- Annual: Fixed capacity subscription from Red Hat

I guess the available options depend on a customer's subscriptions. If I state only to select the "Annual" option, this might not cater for all use cases, such as the trial option. I have subsequently added the following note under the "Select a subscription type step":

> Note: The subscription types that are available to you depend on your OpenShift Dedicated subscriptions and resource quotas.